src/libSchnorr/src/MultiSig.cpp

```diff
@@ -367,7 +367,7 @@ bool MultiSig::MultiSigVerify(const bytes& message, unsigned int offset,
         return false;
       }
       err2 = (BN_nnmod(challenge_built.get(), challenge_built.get(),
-                       Schnorr::GetCurveOrder(), NULL) == 0);
+                       Schnorr::GetCurveOrder(), ctx.get()) == 0);
```

Comment:
This caused a SEGV when running ubuntu 20.04 - the API for bignum indicates that this argument should be provided. It is a scratch variable:

https://linux.die.net/man/3/bn_nnmod

`For all functions, ctx is a previously allocated BN_CTX used for temporary variables; `